src/qililab/platform/platform.py

```diff
@@ -576,11 +576,34 @@ def execute_qprogram(  # pylint: disable=too-many-locals
     ) -> QProgramResults:
         """Execute a QProgram using the platform instruments.
 
+        |
+
+        **The execution is done in the following steps:**
+
+        1. Compile the QProgram.
+        2. Upload the sequences to the instruments.
+        3. Execute the sequences.
+        4. Acquire the results.
+        5. Reset the instrument settings.
```

Comment:
One weird thing I found is the 5th point:
5. Reset the instrument settings.
as this shouldn't be in the execute, more like inside execute_Qblox as it is the source of this last point. I suggest leaving it out and then specifying in each machine what is resetted.

Reply:
If execute_Qblox is an internal method then we should explain here the two clusters Qprogram is able to work with and explain what are we doing on each just here. Maybe two more detailed lists should be better, one for each machine

Reply:
You are right, I'll delete the case concrete bits, and leave it general, 
if you feel like doing the separation, please free and commit it yourself 🙌